src/audio/host.c

```diff
@@ -283,9 +283,9 @@ static int host_trigger(struct comp_dev *dev, int cmd)
 	switch (cmd) {
 	case COMP_TRIGGER_STOP:
 		ret = host_stop(dev);
-		/* host stop fails, let's return the errno */
+		/* host stop fails, let's emit the error log */
 		if (ret)
-			goto out;
+			trace_host_error("TpF");
```

Comment:
Please use more descriptive logs now. e.g. trace_host_error("host: stop failed %d", ret);

Reply:
Good suggestion : )
Actually, my first thought is to use 'trace_error'. But when I went through host.c, I found the whole file doesn't use trace_error like functions. It's only using 3-bytes message. I don't know why and thinking maybe there is some magic reason : - ). I will modify my patch based on your suggestion.